include/aabbox2dSIMD.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __IRR_AABBOX_2D_SIMD_H_INCLUDED__
+#define __IRR_AABBOX_2D_SIMD_H_INCLUDED__
+
+#include "irr/core/math/irrMath.h"
+
+namespace irr { namespace core {
+
+class aabbox2dSIMDf
+{
+public:
+	inline aabbox2dSIMDf()
+		: bounds(-1.0f, -1.0f, 1.0f, 1.0f) {};
```

Comment:
FLT_MAX, otherwise this won't work for boxes with max edge <(1,1) or min edge > (-1,-1)